styles.css

```
@@ -9936,3 +10036,514 @@ body:has(#side-panel.active) #close-fullscreen-cover-btn {
+    line-height: 1.08;
+    letter-spacing: -0.03em;
+}
+
+#fullscreen-cover-overlay #fullscreen-track-artist {
+    margin: 0.12rem 0 0;
+    font-size: 0.94rem;
+    color: rgb(255 255 255 / 0.74);
+}
+
+#fullscreen-cover-overlay #toggle-fullscreen-lyrics-btn,
+#fullscreen-cover-overlay .fullscreen-lyrics-toggle {
```

Stylelint (17.6.0)

[error] Expected selector "#fullscreen-cover-overlay .fullscreen-lyrics-toggle" to come before selector "#fullscreen-cover-overlay.ui-hidden .fullscreen-lyrics-toggle", at line 4112 (no-descending-specificity)

(no-descending-specificity)